doc/release_notes/release_notes_3.0.rst

```diff
@@ -75,7 +140,106 @@ Fixed Issues Details
+- :acrn-issue:`7298` - boot time issue for acrn-dm
+- :acrn-issue:`7297` - update parameter in schema
+- :acrn-issue:`7296` - Segment fault is triggered in course of Virtio-gpu rebooting test
+- :acrn-issue:`7270` - combined cpu_affinity warning for service vm
+- :acrn-issue:`7267` - service vm cpu affinity issue
+- :acrn-issue:`7265` - About 20s after booting uaag with usb mediator,usb disk isn't recognized
+- :acrn-issue:`7261` - Hide PTM in Configurator UI
+- :acrn-issue:`7256` - Remove SCHED_IORR and KERNEL_RAWIMAGE
+- :acrn-issue:`7249` - doc: Exception in Sphinx processing doesn't display error message
+- :acrn-issue:`7248` - restore copyright notice of original author of some files.
+- :acrn-issue:`7246` - Can't fully parse the xml content that was saved by the same version configurator
+- :acrn-issue:`7241` - need copyright notice and license in virtio_gpu.c
+- :acrn-issue:`7212` - Exception"not enough space in guest VE820 SSRAM area" showed when built ACRN with RTCT table
+- :acrn-issue:`7208` - iasl segfault when reboot user VM with virtio-i2c devices
+- :acrn-issue:`7197` - The error message is found after adding the specific mac address in the launch script
+- :acrn-issue:`7172` - [acrn-configuration-tool] offline_cpus won't be executed in NOOP mode
+- :acrn-issue:`7171` - Link to instead of including old release notes in the current release
+- :acrn-issue:`7159` - acrn-config: config tool get_node apic_id failed
+- :acrn-issue:`7136` - [acrn-configuration-tool] Share memory should never support 512M since the HV_RAM_SIZE_MAX is limited to 0x40000000 but not a platform specific problem
+- :acrn-issue:`7133` - Guest VM system reset may fail and ACRN DM program hang
+- :acrn-issue:`7127` - config-tools: remove SERIAL_CONSOLE extracion for bootargs of SOS
+- :acrn-issue:`7124` - [DM]:  Fail to boot the Laag guest if the boot option of "pci=nomsi" is added for Guest kernel
+- :acrn-issue:`7119` - config-tools: bring all cores online
+- :acrn-issue:`7109` - Python traceback if the 'dpkg' tool is not available
+- :acrn-issue:`7098` - Memory leakage bug induced by opendir() in ACRN applications
+- :acrn-issue:`7084` - config_tools:  append passthrough gpu bdf in hexadecimal format
+- :acrn-issue:`7077` - config-tools: find pci hole based on all pci hostbridge
+- :acrn-issue:`7058` - config_tools: board_inspector cannot generate compliable xml for Qemu
+- :acrn-issue:`7045` - [ICX-D] Segmentation fault when passthrough TSN to post_launched VM with enable_ptm option
+- :acrn-issue:`7022` - ACRN debian package not complete when source is not cloned to standard folder
+- :acrn-issue:`7018` - No expected exception generated on some platform.
+
+
 
 Known Issues
 ************
 
+- :acrn-issue:`6631` - [KATA] Kata support is broken in v2.7
+- :acrn-issue:`6978` - [TGL] openstack failed with ACRN v2.7
+- :acrn-issue:`7827` - [Configurator]Pre_launched standard VMs cannot share CPU with Service VM
```

Comment:
Please add this known issue: https://github.com/projectacrn/acrn-hypervisor/issues/7831 @NanlinXie

Reply:
updated.